Strategies/ITVision.md

```diff
@@ -0,0 +1,168 @@
+
+ESDC is narrowing its technology standards focus on ERP, corporate products allowing economies of scale, and production technology stacks for custom built applications only.
+Only on exceptional cases are other technology standardized (e.g. source code version control).
+Product teams manage their own development and testing tools, as well as experimenting with new technology stacks.
+
+Cyber security has shifted to not only include pre-emptive measures but also invest in resilience capabilities.
+IT solutions managing citizen data are built with a [zero trust architecture](https://www.oreilly.com/library/view/zero-trust-networks/9781491962183/ch01.html).
+ESDC's Cyber security has shifted to protecting data at the source and in transit, instead of relying on perimeter defences.
+Encrypted data is treated as unclassified data, therefore increasing the pool of available public cloud services to chose from.
+This involves encrypting data at rest, in transit, and relying on strong authentication, access policies, and modern identity verification to govern access to sensitive data.
+[Behavioral Analytics and biometrics](https://gcn.com/articles/2019/03/25/mobile-verification.aspx) are necessary to secure access to systems and protect citizens from identity theft.
+The [Pan-Canadian Trust Framework](https://canada-ca.github.io/PCTF-CCP/overview/pctf-overview.html) is applied for ESDC's services to enable trusted digital identity to be shared and validated between canadian jurisdictions.
+
+Cyber security is involved from the beginning of IT solution building to the end, and relies on as much automated testing as possible to provide assurance, as opposed to gate-based assessment processes, and relies on embedded knowledge within development teams instead of analysts.
+
+Cyber security manages risks differently depending on which network particular work is being done (production, corporate network, or the Internet) in order to focus efforts more effectively and not hinder operations or innovation.
+
+IT at ESDC adopts modern and agile approaches to development to enable greater adaptability to evolutions in both business requirements and enabling technologies.
+IT solution changes are delivered to production within four months, at most, of a change request.
+ESDC will move towards a micro-services pattern when delivering IT solutions, allowing for greater interoperability and smaller, cheaper, more rapid changes.
+Only through well justified and documented exceptions will IT solutions be allowed to remain as a monolith system.
+
+## 5. A digitally enabled public service
+
+Modern web services (SaaS accessible over the Internet) are opened by default and only on rare occasions, with sound justifications, are they blocked.
+Monitoring of web services is risk managed to focus efforts where risks are perceived and impacts are high.
+If monitoring is required, automation through a combination of Artificial Intelligence disciplines is used to reduce toil.
```

Comment:
If monitoring is required

Monitoring is required ... the question is what kind of monitoring 

Doesn't need to be AI ... could be. Doesn't need to be. Could be automated to some degree without AI, for example. 

Toil should be mentioned somewhere ... not sure if it fits very naturally here

Reply:
As toil is now mentioned in the AI paragraph at the beginning, we could remove this reference.

Reply:
Replaced by:

> If monitoring is required, automation is leveraged through a combination of technologies and disciplines, such Artificial Intelligence.